src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -61,21 +122,36 @@ export default function Page() {
         setCurrentTimesheetsToDisplay (userTimesheets, date); 
     }
     
-    const [userTimesheets, setUserTimesheets] = useState([]); 
-    const [currentTimesheets, setCurrentTimesheets] = useState([]);
-    const [selectedTimesheet, setTimesheet] = useState();
+    //TODO: default to first employee but idk if employee always exists
+    const [selected, setSelected] = useState<UserSchema>();
+    const [user, setUser] = useState<UserSchema>();
+
+    const [userTimesheets, setUserTimesheets] = useState<TimeSheetSchema[]>([]); 
+    const [currentTimesheets, setCurrentTimesheets] = useState<TimeSheetSchema[]>([]);
+    const [selectedTimesheet, setTimesheet] = useState<TimeSheetSchema>();
     const columns = defaultColumns 
 
-    //Pulls user timesheets, marking first returned as the active one
+    // Pulls user timesheets, marking first returned as the active one
     useEffect(() => {
         // Uncomment this if you want the default one loaded 
         //setTimesheet(testingTimesheetResp)
+        apiClient.getUser().then(userInfo => {
```

Comment:
Does this need to be in the use effect here? From my understanding, this is just grabbing the current logged-in user info. Will this ever change (unless they logout, of course), and will it be effected by changes in `selected`? Seems like something that could likely just be grabbed once on page load

Reply:
Good catch, I moved it into a one time useEffect